docs/getting-started.md

```diff
@@ -31,32 +33,60 @@ Theia.
 Theia requires that Antrea v1.7.0 or later is installed in the Kubernetes
 cluster.
 
-For Antrea v1.7, please ensure the Flow Exporter feature of Antrea
-Agent is enabled in the Antrea deployment manifest:
+### Configuration
+
+In addition to enabling the Flow Exporter feature gate, you need to ensure
+that the `flowExporter.enable` flag is set to true in the Antrea Agent
+configuration.
+
+your `antrea-agent` ConfigMap should look like this:
 
 ```yaml
   antrea-agent.conf: |
     ...
     featureGates:
       ...
       FlowExporter: true
+    flowExporter:
+      ...
+      enable: true
 ```
 
 From Antrea v1.8, you can deploy Antrea through Helm by running the following
-commands:
+commands.
+
+When using Antrea v1.13 or greater:
 
 ```bash
 helm repo add antrea https://charts.antrea.io
-helm install antrea antrea/antrea -n kube-system --set featureGates.FlowExporter=true
+helm install antrea antrea/antrea -n kube-system --set featureGates.FlowExporter=true --set flowExporter.enable=true
 ```
 
 This will install the latest available version of Antrea with the Flow Exporter
-feature enabled. You can also install a specific version of Antrea (>= v1.8.0)
+feature enabled.
+
+When using Antrea v1.8 - v1.12:
+
+You can install a specific version of Antrea (v1.8 - v1.12)
 with `--version <TAG>`.
 
+```bash
+helm repo add antrea https://charts.antrea.io
+helm install antrea antrea/antrea -n kube-system --set featureGates.FlowExporter=true --version <TAG>
+```
+
 For more information about Antrea Helm chart, please refer to
 [Antrea Helm chart installation instructions](https://github.com/antrea-io/antrea/blob/main/docs/helm.md).
 
+#### Configuration pre Antrea v1.13
```

Comment:
I changed the structure.

- ### Configuration -> for Antrea v1.13
- #### Configuration pre Antrea v1.13  -> for Antrea v1.8 - v1.12.

Both of them includes the yaml format and helm instruction. Hope it can clear the confusion?